lib/utils/CountryHelper.dart

```diff
@@ -1003,6 +1003,16 @@ extension OpenFoodFactsCoutryExtension on OpenFoodFactsCountry {
   };
 
   String get iso2Code => _ISO_2_CODES[this]!;
+
+  String name() {
+    final String countryName = this
+        .toString()
+        .replaceAll('OpenFoodFactsCountry.', '')
+        .replaceAll('_', ' ');
+    return (countryName.isNotEmpty)
+        ? '${countryName[0].toUpperCase()}${countryName.substring(1).toLowerCase()}'
+        : countryName;
```

Comment:
If name is not empty: name else: name, this doesn't seem logical, does it

Reply:
why not?

Reply:
here, made it easier to read